### PR TITLE
Integrates platform.sh

### PR DESCRIPTION
**Pull request by nvelychenko**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,221 @@
+{
+    "name": "linnovate/openideal-composer",
```
Comment (db-Matroskeen) [inline composer.json:2]: Should we change to "linnovate/openideal-on-platformsh"?

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

**Comment by db-Matroskeen** [comment]
Should we also change the readme file?
Probably we can add a link to the original repository and describe that this template is for platform.sh-hosted projects only.

File: composer.json (hunk 1 above)
Comment (nvelychenko) [reply composer.json:2]: I'm not sure if we need to change it. It's just a repository that fetches changes from linnovate/openideal-composer with the predefined composer.lock file.

**Comment by nvelychenko** [comment]
> Should we also change the readme file?
> Probably we can add a link to the original repository and describe that this template is for platform.sh-hosted projects only.

Done.

**Pull request merged**